suite/tests/api/detach_state.c

```diff
@@ -408,7 +408,94 @@ START_FILE
+        vmovups  [rsp + 21*SIMD_REG_SIZE], zmm21 @N@ \
+        vmovups  [rsp + 22*SIMD_REG_SIZE], zmm22 @N@ \
+        vmovups  [rsp + 23*SIMD_REG_SIZE], zmm23 @N@ \
+        vmovups  [rsp + 24*SIMD_REG_SIZE], zmm24 @N@ \
+        vmovups  [rsp + 25*SIMD_REG_SIZE], zmm25 @N@ \
+        vmovups  [rsp + 26*SIMD_REG_SIZE], zmm26 @N@ \
+        vmovups  [rsp + 27*SIMD_REG_SIZE], zmm27 @N@ \
+        vmovups  [rsp + 28*SIMD_REG_SIZE], zmm28 @N@ \
+        vmovups  [rsp + 29*SIMD_REG_SIZE], zmm29 @N@ \
+        vmovups  [rsp + 30*SIMD_REG_SIZE], zmm30 @N@ \
+        vmovups  [rsp + 31*SIMD_REG_SIZE], zmm31 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 0*OPMASK_REG_SIZE], k0 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 1*OPMASK_REG_SIZE], k1 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 2*OPMASK_REG_SIZE], k2 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 3*OPMASK_REG_SIZE], k3 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 4*OPMASK_REG_SIZE], k4 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 5*OPMASK_REG_SIZE], k5 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 6*OPMASK_REG_SIZE], k6 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 7*OPMASK_REG_SIZE], k7 @N@ \
+        PUSH_GPRS
+#        elif defined(__AVX__)
+#            define PUSHALL                 \
+        lea      rsp, [rsp - NUM_SIMD_REGS*SIMD_REG_SIZE] @N@ \
+        vmovdqu  [rsp +  0*SIMD_REG_SIZE], ymm0 @N@ \
+        vmovdqu  [rsp +  1*SIMD_REG_SIZE], ymm1 @N@ \
+        vmovdqu  [rsp +  2*SIMD_REG_SIZE], ymm2 @N@ \
+        vmovdqu  [rsp +  3*SIMD_REG_SIZE], ymm3 @N@ \
+        vmovdqu  [rsp +  4*SIMD_REG_SIZE], ymm4 @N@ \
+        vmovdqu  [rsp +  5*SIMD_REG_SIZE], ymm5 @N@ \
+        vmovdqu  [rsp +  6*SIMD_REG_SIZE], ymm6 @N@ \
+        vmovdqu  [rsp +  7*SIMD_REG_SIZE], ymm7 @N@ \
+        vmovdqu  [rsp +  8*SIMD_REG_SIZE], ymm8 @N@ \
+        vmovdqu  [rsp +  9*SIMD_REG_SIZE], ymm9 @N@ \
+        vmovdqu  [rsp + 10*SIMD_REG_SIZE], ymm10 @N@ \
+        vmovdqu  [rsp + 11*SIMD_REG_SIZE], ymm11 @N@ \
+        vmovdqu  [rsp + 12*SIMD_REG_SIZE], ymm12 @N@ \
+        vmovdqu  [rsp + 13*SIMD_REG_SIZE], ymm13 @N@ \
+        vmovdqu  [rsp + 14*SIMD_REG_SIZE], ymm14 @N@ \
+        vmovdqu  [rsp + 15*SIMD_REG_SIZE], ymm15 @N@ \
+        PUSH_GPRS
+#        else
+#            define PUSHALL                 \
+        lea      rsp, [rsp - NUM_SIMD_REGS*SIMD_REG_SIZE] @N@ \
+        movups   [rsp +  0*SIMD_REG_SIZE], xmm0 @N@ \
+        movups   [rsp +  1*SIMD_REG_SIZE], xmm1 @N@ \
+        movups   [rsp +  2*SIMD_REG_SIZE], xmm2 @N@ \
+        movups   [rsp +  3*SIMD_REG_SIZE], xmm3 @N@ \
+        movups   [rsp +  4*SIMD_REG_SIZE], xmm4 @N@ \
+        movups   [rsp +  5*SIMD_REG_SIZE], xmm5 @N@ \
+        movups   [rsp +  6*SIMD_REG_SIZE], xmm6 @N@ \
+        movups   [rsp +  7*SIMD_REG_SIZE], xmm7 @N@ \
+        movups   [rsp +  8*SIMD_REG_SIZE], xmm8 @N@ \
```

Comment:
Ditto

Reply:
Same response.

Reply:
(You can see all this code is using "rsp" too, so it's not just these extra registers that would fail for 32-bit.)